h5p_proplist.go

```diff
@@ -40,6 +44,26 @@ func (p *PropList) Close() error {
 	return p.closeWith(h5pclose)
 }
 
+// Sets the size of the chunks used to store a chunked layout dataset.
```

Comment:
please prepend `SetChunk` in that sentence.

Reply:
Fixed